src/app/delegation_backend/src/delegation_backend/app_config.go

```diff
@@ -0,0 +1,98 @@
+package delegation_backend
+
+import (
+	"encoding/json"
+	"os"
+
+	logging "github.com/ipfs/go-log/v2"
+)
+
+func loadAwsCredentials(filename string, log logging.EventLogger) {
+	file, err := os.Open(filename)
+	if err != nil {
+		log.Errorf("Error loading credentials file: %s", err)
+		os.Exit(1)
+	}
+	defer file.Close()
+	decoder := json.NewDecoder(file)
+	var credentials AwsCredentials
+	err = decoder.Decode(&credentials)
+	if err != nil {
+		log.Errorf("Error loading credentials file: %s", err)
+		os.Exit(1)
+	}
+	os.Setenv("AWS_ACCESS_KEY_ID", credentials.AccessKeyId)
+	os.Setenv("AWS_SECRET_ACCESS_KEY", credentials.SecretAccessKey)
+}
+
+func LoadEnv(log logging.EventLogger) AppConfig {
+	var config AppConfig
+
+	configFile := os.Getenv("CONFIG_FILE")
+	if configFile != "" {
+		file, err := os.Open(configFile)
+		if err != nil {
+			log.Errorf("Error loading config file: %s", err)
+			os.Exit(1)
```

Comment:
I'm not a go expert but if we are closing a file in line 38, shouldn't we do the same before exiting in line 36?

Reply:
This is equivalent to calling `log.Fatalf`. I should probably uniform the code to use the same approach everywhere.